### PR TITLE
Random number generator and math operations are ready!

### PR DESCRIPTION
**Pull request by Alexandr-R-94**

### CODE REVIEW

**Review by Alex99905 — APPROVED** [review]

File: homework/java/alexandr_raby/MathOperation/aTemplate/src/main/java/by/jrr/start/MahOperationDemo.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+        int a = 90;
+        int b = 40;
+
+        System.out.println("1) Addition '+':");
+        System.out.print("a = ");
+        System.out.print(a);
+        System.out.print(", ");
+        System.out.print("b = ");
+        System.out.print(b);
```
Comment (LiudmilaZhdanok) [inline homework/java/alexandr_raby/MathOperation/aTemplate/src/main/java/by/jrr/start/MahOperationDemo.java:16]: if you write "println" there you don't need to write "System.out.println()" later.

File: homework/java/alexandr_raby/MathOperation/aTemplate/src/main/java/by/jrr/start/MahOperationDemo.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+        int sum = a + b;
+
+        System.out.println();
+        System.out.print("a + b = ");
+        System.out.print(a);
+        System.out.print(" + ");
+        System.out.print(b);
+        System.out.print(" = ");
+        System.out.print(sum);
```
Comment (LiudmilaZhdanok) [inline homework/java/alexandr_raby/MathOperation/aTemplate/src/main/java/by/jrr/start/MahOperationDemo.java:26]: if you write "println" there you  need to write "System.out.println()" only once later.

File: homework/java/alexandr_raby/MathOperation/aTemplate/src/main/java/by/jrr/start/MahOperationDemo.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+
+        double c = 10.18;
+        double d = 2.14;
+
+        System.out.println();
+        System.out.println();
+        System.out.println("2) Substraction '-':");
+        System.out.print("c = " + c +", d = " + d);
```
Comment (LiudmilaZhdanok) [inline homework/java/alexandr_raby/MathOperation/aTemplate/src/main/java/by/jrr/start/MahOperationDemo.java:35]: if you write "println" there you don't need to write "System.out.println()" later.

File: homework/java/alexandr_raby/MathOperation/aTemplate/src/main/java/by/jrr/start/MahOperationDemo.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+        double dif = c - d;
+
+        System.out.println();
+        System.out.print("c - d = " + c + " - " + d + " = " + dif);
```
Comment (LiudmilaZhdanok) [inline homework/java/alexandr_raby/MathOperation/aTemplate/src/main/java/by/jrr/start/MahOperationDemo.java:40]: if you write "println" there you need to write "System.out.println()" only once later.

**Review by zhdanok — APPROVED** [review]
See notes, please

**Review by zhdanok — APPROVED** [review]
lgfm

**Review by zhdanok — APPROVED** [review]
lgfm

**Review by DenisAbramovDm — APPROVED** [review]
Lgfm

**Review by zhdanok — APPROVED** [review]
Lgfm

**Pull request merged**